### DISCUSSION TITLE
Please publish the releases on https://github.com/encode/django-rest-framework/releases

### DISCUSSION THREAD

**Discussion by kevinrenskers** [encode/django-rest-framework#9346] [General]
There is no top-level changelog.md in the repo itself, and https://github.com/encode/django-rest-framework/releases is missing the last two releases. You have to go to https://www.django-rest-framework.org/community/release-notes/ to view the changelog, or dig through the repo to find https://github.com/encode/django-rest-framework/blob/master/docs/community/release-notes.md.

It would be really nice to publish the last two releases on https://github.com/encode/django-rest-framework/releases so there's an easily accessible place to view all release notes. Thanks!

**Comment by jaap3** [comment]
I was about to raise this. It is also helpful for projects that have dependabot enabled, as it uses the GitHub release to show changes in the update PR.
